cmd/osctl/cmd/cluster.go

```diff
@@ -212,10 +165,22 @@ func create(ctx context.Context) (err error) {
 	// Create the master nodes.
 	for i := 0; i < masters; i++ {
 		var typ generate.Type
+
+		var configDataStruct *v1alpha1.Config
+
+		var configDataString string
+
 		if i == 0 {
 			typ = generate.TypeInit
+			configDataStruct = configBundle.InitCfg
 		} else {
 			typ = generate.TypeControlPlane
+			configDataStruct = configBundle.ControlPlaneCfg
+		}
+
+		configDataString, err = configDataStruct.String()
+		if err != nil {
+			return err
```

Comment:
Not something for this PR, but I think we can get this to use the interface methods. I would like to tackle that just to get a feel for things if you don't mind?

Reply:
oh yeah, we should be getting those with the methods for sure. whoops. I'll let you fix after we merge.